release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -1120,6 +1120,13 @@ go-template={{.status.latestVersion}` instead.
 Container Native Storage (CNS) is now called Red Hat OpenShift Container Storage
 (RHOCS). Previously, there was confusion between CNS and CRS terminology.
 
+[discrete]
+[[ocp-310-builder-image-replaced]]
+=== Builder Image Replaced
+
+In {product-title} 3.10, the `registry.access.redhat.com/openshift3/ose-docker-builder`
+image replaced the `registry.access.redhat.com/openshift3/ose-sti-builder` image. 
```

Comment:
might be confusing since the docker-builder image already existed, it's just now that it fulfills both roles where before it only did one and sti-builder did the other.

Reply:
Thanks, @bparees! Hopefully this is a bit better.